src/jmh/java/dev/blaauwendraad/masker/json/BenchmarkUtils.java

```diff
@@ -15,6 +22,25 @@
 
 public class BenchmarkUtils {
 
+    public static final long STATIC_RANDOM_SEED = 1285756302517652226L;
+    public static final int TARGET_KEYS_SIZE = 2385;
```

Comment:
Very specific

Reply:
It's the number of keys we have in target_keys.json, I just needed the precise number somewhere